lib/wpdb_activerecord/post.rb

```diff
@@ -17,5 +17,17 @@ class Post < ActiveRecord::Base
     has_many :attachments, -> { where(post_type: "attachment") }, foreign_key: "post_parent", class_name: WPDB.configuration.post_class
     has_many :revisions, -> { where(post_type: "revision") }, foreign_key: "post_parent", class_name: WPDB.configuration.post_class
     has_many :postmetas, foreign_key: "post_id", class_name: WPDB.configuration.postmeta_class
+
+    private
+
+    if Gem::Version.new(Rails::VERSION::STRING) >= Gem::Version.new("5.1.0")
```

Comment:
You can simply use `Rails.gem_version` to get current Rails version as `Gem::Version` instance